dencam/mppt.py

```diff
@@ -1,3 +1,5 @@
+'''Handles the data import from the sunsaver mppt controller
+and formats the data to be displayed on the screen'''
```

Comment:
Docstrings should be triple double quotes.  And the second set should be on a line by themselves for a mult-line docstring.  See PEP 257 for convention: [https://peps.python.org/pep-0257](url)